contracts/modules/capital/MCR.sol

```diff
@@ -152,37 +141,23 @@ contract MCR is IMCR, MasterAwareV2 {
       return;
     }
 
-    if (block.timestamp > _lastUpdateTime && pool().calculateMCRRatio(poolValueInEth, _mcr) >= _mcrFloorIncrementThreshold) {
-        // MCR floor updates by up to maxMCRFloorIncrement percentage per day whenever the MCR ratio exceeds 1.3
-        // MCR floor is monotonically increasing.
-      uint basisPointsAdjustment = Math.min(
-        _maxMCRFloorIncrement * (block.timestamp - _lastUpdateTime) / 1 days,
-        _maxMCRFloorIncrement
-      );
-      uint newMCRFloor = _mcrFloor * (basisPointsAdjustment + BASIS_PRECISION) / BASIS_PRECISION;
-      require(newMCRFloor <= type(uint112).max, 'MCR: newMCRFloor overflow');
-
-      mcrFloor = uint112(newMCRFloor);
-    }
-
     // sync the current virtual MCR value to storage
-    uint112 newMCR = uint112(getMCR());
+    uint80 newMCR = getMCR().toUint80();
     if (newMCR != _mcr) {
       mcr = newMCR;
     }
 
-    // the desiredMCR cannot fall below the mcrFloor but may have a higher or lower target value based
-    // on the changes in the totalSumAssured in the system.
-    uint totalSumAssured = getAllSumAssurance();
+    uint totalSumAssured = getTotalActiveCoverAmount();
```

Comment:
nit picking, but we could rename `totalSumAssured` to `totalActiveCoverAmount` now